### PR TITLE
[doc][cdc] Updated diagrams

### PR DESCRIPTION
**Pull request by ddhodge**
Updated diagrams

@netlify /preview/architecture/docdb-replication/change-data-capture/

### CODE REVIEW

**Comment by siddharth2411** [comment]
Reviewed the architecture diagram, it looks good to me. Would defer to @yogesh-yb for approval since there are other changes.

**Review by ymahajan — APPROVED** [review]
Looks good to me.

File: docs/content/preview/architecture/docdb-replication/change-data-capture.md [hunk 1]
```diff
@@ -39,11 +37,13 @@ The Debezium YugabyteDB connector captures row-level changes in the schemas of a
...
+
+Creating a new CDC stream returns a stream UUID. This is facilitated via the [yb-admin](../../../admin/yb-admin/#change-data-capture-cdc-commands) tool. A stream ID is created first, per database. You configure the maximum batch side in YugabyteDB, while the polling frequency is configured on the connector side.
 
-The core primitive of CDC is the _stream_. Streams can be enabled and disabled on databases. Every change to a watched database table is emitted as a record in a configurable format to a configurable sink. Streams scale to any YugabyteDB cluster independent of its size and are designed to impact production traffic as little as possible.
+Connector tasks can consume changes from multiple tablets. At least once delivery is guaranteed. In turn, connector tasks write to the Kafka cluster, and tasks don't need to match Kafka partitions. Tasks cab be independently scaled up or down.
```
Comment (aishwarya24) [inline docs/content/preview/architecture/docdb-replication/change-data-capture.md:44]:
```suggestion
Connector tasks can consume changes from multiple tablets. At least once delivery is guaranteed. In turn, connector tasks write to the Kafka cluster, and tasks don't need to match Kafka partitions. Tasks can be independently scaled up or down.
```

File: docs/content/preview/architecture/docdb-replication/change-data-capture.md [hunk 2]
```diff
@@ -39,11 +37,13 @@ The Debezium YugabyteDB connector captures row-level changes in the schemas of a
...
 
-![How does CDC work](/images/explore/cdc-overview-work3.png)
+The connector produces a change event for every row-level insert, update, and delete operation that was captured, and sends change event records for each table in a separate Kafka topic. Client applications read the Kafka topics that correspond to the database tables of interest, and can react to every row-level event they receive from those topics. For each table, the default behavior is that the connector streams all generated events to a separate Kafka topic for that table. Applications and services consume data change event records from that topic. All changes for a row (or rows in the same tablet) are received in the in the order in which they happened. A checkpoint per stream ID and tablet is updated in a state table after a successful write to Kafka brokers.
```
Comment (aishwarya24) [inline docs/content/preview/architecture/docdb-replication/change-data-capture.md:46]:
```suggestion
The connector produces a change event for every row-level insert, update, and delete operation that was captured, and sends change event records for each table in a separate Kafka topic. Client applications read the Kafka topics that correspond to the database tables of interest, and can react to every row-level event they receive from those topics. For each table, the default behavior is that the connector streams all generated events to a separate Kafka topic for that table. Applications and services consume data change event records from that topic. All changes for a row (or rows in the same tablet) are received in the order in which they happened. A checkpoint per stream ID and tablet is updated in a state table after a successful write to Kafka brokers.
```

**Review by aishwarya24 — APPROVED** [review]
LGTM

**Pull request merged**